lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "printer"
+require_relative "order"
+
+class Menu
+
+  attr_reader :food_options, :printer
```

Comment:
Do food_options and printer need to be publicly available on the Menu? Think these could be private attributes